themes/bootstrap3/templates/Auth/MultiILS/loginfields.phtml

```diff
@@ -4,7 +4,7 @@
   <select id="login_target" name="target" class="form-control">
     <?foreach ($this->auth()->getManager()->getLoginTargets() as $target):?>
```

Comment:
Demian, could be this the problem? The opening tag is not followed by space and then not catched by this patch.

Reply:
I just looked into the first file reported by Travis, the other files could have the same or other problem ;)